### DISCUSSION TITLE
8.11 树形动态规划

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#108] [Comments]
# 8.11 树形动态规划

1. 树形动态规划简介 树形动态规划：简称为「树形 DP」，是一种在树形结构上进行推导的动态规划方法。如下图所示，树形 DP 的求解过程一般以节点从深到浅（子树从小到大）的顺序作为动态规划的「阶段」。在树形 DP 中，第 $1$ 维通常是节点编号，代表以该节点为根的子树。 树形 DP 问题的划分方法有多种方式。 如果按照「阶段转移的方向」进行划分，可以...

[https://algo.itcharge.cn/08_dynamic_programming/08_11_tree_dp/](https://algo.itcharge.cn/08_dynamic_programming/08_11_tree_dp/)

**Comment by sadcoldpig** [comment]
3.2 写的不如 参考资料1，没有体现换根

**Reply by itcharge** [reply]
感谢您的建议，后续会努力改进文章内容的
